package-lock.json

```diff
@@ -1,9327 +0,0 @@
-{
-  "name": "arc-ui-app",
```

Comment:
Package-lock has been deleted but doesn't appear to have been brought back. Please readd package-lock

Reply:
Somehow it was in the .gitignore, removed and added to repo